docs/_config.yml

```diff
@@ -0,0 +1,48 @@
+# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+
+# To test documentation changes locally or using GitHub Pages, see:
+# https://github.com/coreos/fedora-coreos-tracker/blob/main/docs/testing-project-documentation-changes.md
+
+title: {{ pretty_name }}
+description: {{ pretty_name }} documentation
+baseurl: "/{{ git_repo }}"
+url: "https://coreos.github.io"
+permalink: /:title/
+markdown: kramdown
+kramdown:
+  typographic_symbols:
+    ndash: "--"
+    mdash: "---"
+
+remote_theme: just-the-docs/just-the-docs@v0.4.1
+plugins:
+  - jekyll-remote-theme
+
+color_scheme: coreos
+
+# Aux links for the upper right navigation
+aux_links:
+  "See on GitHub":
```

Comment:
I think `{{ pretty_name }} on GitHub` was a bit cleaner, but this is okay.

Reply:
Sure, fixed.